doc/developer/scratch.md

```diff
@@ -124,5 +126,87 @@ bin/scratch destroy --all-mine
 Pass `--dry-run` if you want to see what instances `bin/scratch destroy` would
 destroy without actually destroyin them.
 
+### Grafana integration
+
+The `materialized` process always exposes metrics at via its primary port's http server on the
+prometheus-standard `/metrics` path, but the deployed server needs to be configured correctly for
```

Comment:
```suggestion
prometheus-standard `/metrics` path, but the scratch instance needs to be configured correctly for
```

(Wasn't clear to me whether "server" referred to the just-mentioned `materialized` process or the scratch instance; I think this is what you meant so I figured I'd make it explicit.)